Add chat model and notifications model

sangwoo-jeong opened the pull request:
## ✅ PR 요약
- 관련 이슈 번호: 모델 작성
- 작업 요약: App 폴더 생성과 모델 작성 완료 및 마이그레이션

## 📄 상세 내용
- [x] App 폴더 생성
- [x] models 작성 완료
- [x] 마이그레이션 완료
- [x] chat models에서 content의 max_length를 500자로 지정.
- [x] notifications에서 back_link_url의 길이를 2048자에서 255자로 변경 및
  notification_type의 max_length를 20자로 최적화
- [x] admin파일 모두 제거

## 📸 스크린샷 (선택)
<img width="1184" height="691" alt="image" src="https://github.com/user-attachments/assets/a657bc5e-35cb-4f12-9659-268169190dcd" />

## 📝 기타 참고 사항
urls와 views는 본격적 개발 단계에서 생성할 예정 입니다. 

## 🧪 PR Checklist
- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

sangwoo-jeong committed:
added chat and notifications to installed app

sangwoo-jeong committed:
✨ Feat: add chat app and migration

모델도 작성 완료

sangwoo-jeong committed:
✨ Feat: add notifications app and migration

모델까지 작성 완료 됨

sangwoo-jeong committed:
🎨 Ran black/isort/mypy

못 돌렸던 코드 포매터 실행 완료

Meoyoug commented on apps/chat/admin.py:
장고 어드민 사용하지 않을 거라 이건 제거해주셔도 됩니당

Meoyoug commented on apps/notifications/admin.py:
이 어드민도 제거해주세용

Meoyoug commented on apps/notifications/models.py:
```diff
@@ -0,0 +1,34 @@
+    content = models.CharField(max_length=300)
+    notification_type = models.CharField(max_length=30, choices=NotificationType.choices)
+    is_read = models.BooleanField(default=False)
+    back_url_link = models.CharField(max_length=2048)  # URL은 길어질 수 있으므로
```
음..url길이가 길어진다 한들 2048자까지는안될거같은데요..?
255자 내도 충분하지 않을까 싶어요

Umdoong replied:
동의합니다

Meoyoug commented on apps/chat/models.py:
```diff
@@ -0,0 +1,34 @@
+        on_delete=models.CASCADE,
+        related_name="chat_messages",
+    )
+    content = models.TextField()
```
이거 그냥 필드에서 max_length 추가해주세요!

sangwoo-jeong committed:
🐛 fix: chat과 notifications의 models 수정

- chat models에서 content의 max_length를 500자로 지정.
- notifications에서 back_link_url의 길이를 2048자에서 255자로 변경 및
  notification_type의 max_length를 20자로 최적화
- admin파일 모두 제거

Umdoong reviewed (dismissed):
확인했습니다.

Meoyoug reviewed (dismissed):
고생하셨습니다 머지할게용

sangwoo-jeong committed:
🐛 fix: notifications에서의 upcomming 타이포 수정

알림 모델에서의 오타를 수정하였습니다. (upcomming->upcoming)

Meoyoug reviewed (approved):
찐으로 고생하셨습니다.
